app/assets/javascripts/new_admin/navbar.js

```diff
@@ -51,7 +51,7 @@ tabButtons.forEach((buttonElement) => {
     setActiveButton(buttonElement)
   });
 });
-tabButtons[0].click();
+tabButtons[0]?.click();
```

Comment:
when the tab buttons will be null?

Reply:
It seems that `Cannot read properties of undefined` is raised when no `data-tab-id` is found in a NewAdmin page.
The following example is from the `show` Project page, when no revenue forecast or allocation is found for the specific project, meaning, in this scenario, no `data-tab-id` (or `data-tab-content`):
<img width="1438" alt="Captura de Tela 2023-08-07 às 10 47 32" src="https://github.com/Codeminer42/Punchclock/assets/52546576/157c3bbd-34e1-47b8-bc2d-fbd0599fbf9b">